### PR TITLE
[APM] Update PHP tracer docs with the latest env vars

### PR DESCRIPTION
**Pull request by SammyK**
### What does this PR do?

Also add a table of default service names for each integration.

### Preview link

https://docs-staging.datadoghq.com/sammyk/php-env-vars/tracing/setup/php/

### CODE REVIEW

File: content/en/tracing/setup/php.md [hunk 1]
```diff
@@ -234,28 +235,60 @@ DD_TRACE_DEBUG=true php -S localhost:8888
...
-| `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED`  | `false`     | Enable URL's as resource names; [see "Map Resource Names To Normalized URI"](#map-resource-names-to-normalized-uri)                            |
-| `DD_<INTEGRATION>_ANALYTICS_ENABLED`      | `false`     | Flag to enable app analytics for relevant spans in a specific integration                                                                      |
-| `DD_SERVICE_MAPPING`      | `null`     | Change the default name of an APM integration. Rename one or more integrations at a time, for example: `DD_SERVICE_MAPPING=pdo:payments-db,mysqli:orders-db`                                                                      |
+| `DD_TRACE_RESOURCE_URI_MAPPING`           | `null`      | CSV of URL-to-resource-name mapping rules. (e.g., `/foo/*,/bar/$*/baz`; [see "Custom URL-To-Resource Mapping"](#custom-url-to-resource-mapping)) |
+| `DD_TRACE_SAMPLE_RATE`                    | `1.0`       | The sampling rate for the traces. Between `0.0` and `1.0` (default). It was `DD_SAMPLING_RATE` before v0.36.0                                  |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/php.md:260]:
```suggestion
| `DD_TRACE_SAMPLE_RATE`                    | `1.0`       | The sampling rate for the traces (defaults to: between `0.0` and `1.0`). For versions <0.36.0, this parameter is `DD_SAMPLING_RATE`.                                  |
```

File: content/en/tracing/setup/php.md [hunk 2]
```diff
@@ -234,28 +235,60 @@ DD_TRACE_DEBUG=true php -S localhost:8888
...
+| `DD_TRACE_SAMPLING_RULES`                 | `null`      | A JSON encoded string to configure the sampling rate. (e.g. Set the sample rate to 20%: `[{"sample_rate": 0.2}]`. Set the sample rate to 10% for services starting with 'a' and span name 'b'. Set the sample rate to 20% for all other services: `[{"service": "a.*", "name": "b", "sample_rate": 0.1}, {"sample_rate": 0.2}]`); [see "Integration Names"](#integration-names) |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/php.md:261]:
```suggestion
| `DD_TRACE_SAMPLING_RULES`                 | `null`      | A JSON encoded string to configure the sampling rate. Examples: Set the sample rate to 20%: `[{"sample_rate": 0.2}]`. Set the sample rate to 10% for services starting with 'a' and span name 'b' and set the sample rate to 20% for all other services: `[{"service": "a.*", "name": "b", "sample_rate": 0.1}, {"sample_rate": 0.2}]` (see [Integration names](#integration-names)). |
```

File: content/en/tracing/setup/php.md [hunk 3]
```diff
@@ -234,28 +235,60 @@ DD_TRACE_DEBUG=true php -S localhost:8888
...
+| `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED`  | `true`      | Enable URL's as resource names; [see "Map Resource Names To Normalized URI"](#map-resource-names-to-normalized-uri)                            |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/php.md:262]:
```suggestion
| `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED`  | `true`      | Enable URL's as resource names (see [Map resource names to normalized URI](#map-resource-names-to-normalized-uri)).                            |
```

File: content/en/tracing/setup/php.md [hunk 4]
```diff
@@ -234,28 +235,60 @@ DD_TRACE_DEBUG=true php -S localhost:8888
...
+| `DD_<INTEGRATION>_ANALYTICS_ENABLED`      | `false`     | Flag to enable app analytics for relevant spans in a specific integration; [see "Integration Names"](#integration-names)                       |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/php.md:263]:
```suggestion
| `DD_<INTEGRATION>_ANALYTICS_ENABLED`      | `false`     | A flag to enable app analytics for relevant spans in a specific integration (see [Integration names](#integration-names)).                       |
```

File: content/en/tracing/setup/php.md [hunk 5]
```diff
@@ -234,28 +235,60 @@ DD_TRACE_DEBUG=true php -S localhost:8888
...
+| `DD_<INTEGRATION>_ANALYTICS_SAMPLE_RATE`  | `1.0`       | Set the app analytics sample rate for relevant spans in a specific integration; [see "Integration Names"](#integration-names)                  |
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/php.md:264]:
```suggestion
| `DD_<INTEGRATION>_ANALYTICS_SAMPLE_RATE`  | `1.0`       | Set the app analytics sample rate for relevant spans in a specific integration (see [Integration names](#integration-names)).                  |
```

File: content/en/tracing/setup/php.md [hunk 6]
```diff
@@ -234,28 +235,60 @@ DD_TRACE_DEBUG=true php -S localhost:8888
...
+
+#### Integration names
+
+The table below specifies the default service names for each integration. Change the service names with `DD_SERVICE_MAPPING`.
+
+Use the name when setting integration-specific configuration such as, `DD_<INTEGRATION>_ANALYTICS_ENABLED`. e.g. Laravel would be `DD_LARAVEL_ANALYTICS_ENABLED`.
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/php.md:270]:
```suggestion
Use the name when setting integration-specific configuration such as, `DD_<INTEGRATION>_ANALYTICS_ENABLED`, for example: Laravel is `DD_LARAVEL_ANALYTICS_ENABLED`.
```

File: content/en/tracing/setup/php.md [hunk 7]
```diff
@@ -234,28 +235,60 @@ DD_TRACE_DEBUG=true php -S localhost:8888
...
+| Mysqli            | `mysqli`          |
+| PDO               | `pdo`             |
+| Predis            | `predis`          |
+| Slim              | `slim`            |
+| Symfony           | `symfony`         |
+| WordPress         | `wordpress`       |
+| Yii               | `yii`             |
+| ZendFramework     | `zendframework`   |
 
 #### Map Resource Names To Normalized URI
```
Comment (ruthnaebeck) [inline content/en/tracing/setup/php.md:293]:
```suggestion
#### Map resource names to normalized URI
```

**Comment by SammyK** [comment]
Thanks for the review @ruthnaebeck! Suggested changes have been applied.

**Pull request merged**